pocs/tests/test_pocs.py

```diff
@@ -385,8 +390,7 @@ def test_power_down_while_running(pocs):
     assert pocs.connected is False
 
 
-def test_power_down_dome_while_running(pocs_with_dome):
```

Comment:
You need to add these two lines back in for the three failing tests and then they should pass.